contrib/docker/solr/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN mkdir -p /opt/solr/server/solr/$SOLR_CORE/data
 
 # Adding Files
 ADD ./solrconfig.xml \
-https://raw.githubusercontent.com/ckan/ckan/ckan-2.5.2/ckan/config/solr/schema.xml \
+https://raw.githubusercontent.com/ckan/ckan/master/ckan/config/solr/schema.xml \
```

Comment:
or a simpler idea: change this to /dev-v2.6/ instead of /master/.  We don't change the solr schema very often and that will keep old versions of this code building the right solr image.